semantic/graph.go

```diff
@@ -220,8 +220,9 @@ func (p *PackageClause) Copy() Node {
 type ImportDeclaration struct {
 	Loc
 
-	As   *Identifier
-	Path *StringLiteral
+	As       *Identifier
+	Registry *string
```

Comment:
Nit: When resolving the import here on the go side you could add an error case for when a registry exist, to serve as a point to fill in the actual implementation later if nothing else.

Reply:
This is going to follow as the next PR for the Go side.